src/plugins/intel_cpu/src/nodes/gather_elements.cpp

```diff
@@ -37,22 +37,20 @@ GatherElements::GatherElements(const std::shared_ptr<ov::Node>& op, const GraphC
     if (!isSupportedOperation(op, errorMessage)) {
         OPENVINO_THROW_NOT_IMPLEMENTED(errorMessage);
     }
-    errorPrefix_ = std::string("Layer GatherElements with name '") + op->get_friendly_name() + "'";
-
     if (inputShapes.size() != 2 || outputShapes.size() != 1)
-        OPENVINO_THROW(errorPrefix_, " has invalid number of input/output edges.");
+        THROW_CPU_NODE_ERR(" has invalid number of input/output edges.");
```

Comment:
```suggestion
        THROW_CPU_NODE_ERR("has invalid number of input/output edges.");
```
We don't need space at the beginning of the error message here. Here and below.

Reply:
Removed spaces in such locations